src/objects/player.gd

```diff
@@ -28,6 +28,7 @@ var is_vulnerable := true
 var last_footstep_pos: Vector3 = Vector3.ZERO
 var is_drawing_back := false
 var drawback_time := 0.0
+var quiver := 1
```

Comment:
rename to something like quiver_count. Just `quiver` makes me think it's an object